regolith/main.py

```diff
@@ -269,9 +283,7 @@ def main(args=None):
     if "schemas" in rc._dict:
         user_schema = copy.deepcopy(rc.schemas)
         rc.schemas = copy.deepcopy(SCHEMAS)
-        for k in user_schema:
-            for k2, v in user_schema[k].items():
-                rc.schemas[k][k2].update(v)
+        merge_nested_dicts(rc.schemas, user_schema)  # merge user schemas into the default schemas
```

Comment:
I think we don't need the comment.  It can be deleted.

Reply:
done.